.github/config.yml

```diff
@@ -20,6 +20,6 @@ firstPRMergeComment: >
 
   <a href="https://www.jenkins.io/participate/" target="_blank">
    <picture>
-     <img width="600" src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
+     <img width="600" alt="jenkins welcome page" src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
```

Comment:
Jenkins is an English language proper noun.  It needs to be capitalized.
```suggestion
     <img width="600" alt="Jenkins welcome page" src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
```

Reply:
Needs to be localizable.

```suggestion
     <img width="600" alt="${%Jenkins welcome page}" src="https://raw.githubusercontent.com/jenkinsci/jenkins/master/.github/images/jenkins-welcome.svg">
```

Additionally I am not sure _welcome page_ adequately describes what this is.

Reply:
I don't think that we can localize the text in pages that are generated by GitHub Actions.

Reply:
Oops 👍